test/common/middleware/ChaosHandler.ts

```diff
@@ -276,5 +276,17 @@ describe("ChaosHandler.ts", () => {
 			};
 			assert.isDefined(tempChaosHandlerManual["execute"](cxt));
 		});
+
+		it("Should return response for Manual Request Level case 100%", async () => {
```

Comment:
no await in the body, remove the async

Reply:
They are all using async without await... Want me to clear all of them in this file?

Reply:
sure. thanks to taking this on.